src/components/App/App.js

```diff
@@ -38,23 +39,24 @@ class App extends Component {
   render() {
     return (
       
-        <React.Fragment>
+        <main>
           <nav>
             <h1>Rancid Tomatillos</h1>
           </nav>
-          {(!this.state.movies.length && !this.state.errorMessage) && <Loading /> }
-          {this.state.errorMessage && <h2 className='error-message'> {this.state.errorMessage} </h2>}
         
             <Route exact path='/'>
+              {(!this.state.movies.length && !this.state.errorMessage) && <Loading /> }
+              {this.state.errorMessage &&  <Error />}
+              {//<h2 className='error-message'> {this.state.errorMessage} </h2> 
```

Comment:
Since we have a component for the error now we should delete this commented out code